pkgs/servers/endlessh-go/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildGoModule, fetchFromGitHub, lib, ... }:
+
+buildGoModule rec {
+  pname = "endlessh-go";
+  version = "20220213";
+
+  src = fetchFromGitHub {
+    owner = "shizunge";
+    repo = "endlessh-go";
+    rev = version;
+    sha256 = "sha256-x/38w0GtzYBGWr0ZkfY2HmDEAUI54R833aH0RZSCTC0=";
+  };
+  vendorSha256 = "sha256-h/DpbXO+LUsB9NOAXUfNx3VOfEsiolfBEMBrAqVlU3A=";
+  proxyVendor = true;
```

Comment:
Do we really need this?

Reply:
This won't build otherwise. If I'm understanding `buildGoModule` correctly, `proxyVendor` should be applied here because there is no other mechanism for dealing with dependencies in the source repo.